ui-v2/src/hooks/global-concurrency-limits.ts

```diff
@@ -47,11 +48,19 @@ export const buildListGlobalConcurrencyLimitsQuery = (
 /**
  *
  * @param filter
- * @returns list of global concurrency limits as a QueryResult object
+ * @returns list of global concurrency limits as a SuspenseQueryResult object
  */
+
 export const useListGlobalConcurrencyLimits = (
-	filter: GlobalConcurrencyLimitsFilter,
-) => useQuery(buildListGlobalConcurrencyLimitsQuery(filter));
+	filter: GlobalConcurrencyLimitsFilter = { offset: 0 },
+) => useSuspenseQuery(buildListGlobalConcurrencyLimitsQuery(filter));
+
+useListGlobalConcurrencyLimits.loader = ({
```

Comment:
ended up using this `.loader` style for consistency with `Variable`.

But some principal I'm trying to keep in mind is 

>  How do we keep our React queries reusable throughout the webapp?

I _think_ just keeping it as queryOptions being exposed to the webapp gives this flexibility because you can feed it into different loaders, `useQuery`, `useQueries`, `useSuspenseQuery`, and `useSuspenseQueries` for different scenarios.

**But** let keep building more stuff to see some similar patterns